recipes-demos/ti-apps-launcher/ti-apps-launcher.bb

```diff
@@ -1,4 +1,4 @@
-PR = "r17"
+PR = "r18"
```

Comment:
Have we created a PR on ti-apps-launcher sources to add terminal icons in menu?

If not, We might need another PR to handle the SRCREV update of ti-apps-launcher to include the Terminal icons in menu bar.

Reply:
I will create a separate PR to update SRCREV later. We might skip updating PR in that commit as its a small change.